oshi-core/src/main/java/oshi/software/os/mac/MacOperatingSystem.java

```diff
@@ -119,52 +121,11 @@ public FamilyVersionInfo queryFamilyVersionInfo() {
 
     private String parseCodeName() {
         if (this.major == 10) {
-            /*
-            switch (this.minor) {
-            // MacOS
-            case 15:
-                return "Catalina";
-            case 14:
-                return "Mojave";
-            case 13:
-                return "High Sierra";
-            case 12:
-                return "Sierra";
-            // OS X
-            case 11:
-                return "El Capitan";
-            case 10:
-                return "Yosemite";
-            case 9:
-                return "Mavericks";
-            case 8:
-                return "Mountain Lion";
-            case 7:
-                return "Lion";
-            case 6:
-                return "Snow Leopard";
-            case 5:
-                return "Leopard";
-            case 4:
-                return "Tiger";
-            case 3:
-                return "Panther";
-            case 2:
-                return "Jaguar";
-            case 1:
-                return "Puma";
-            case 0:
-                return "Cheetah";
-            default:
-            }
-            */
-            final String OSHI_VERSIONS_PROPERTIES = "oshi.versions.properties";
-            Properties verProps = FileUtil.readPropertiesFromFilename(OSHI_VERSIONS_PROPERTIES);
-
-            String major = String.valueOf(this.major);
-            String minor = String.valueOf(this.minor);
-            String version_number = major+"."+minor;
             
+            MACOS_VERSIONS_PROPERTIES = "macos.versions.properties";
```

Comment:
Sorry I wasn't more clear here.  Need to keep the `oshi` to prevent conflicting with any other possible properties files on the class path.   But define the string `final` up above instead of changing it here.

Reply:
Thank you so much for being with at each step @dbwiddis , i am on it to make the changes.